tests/find_cmd_tests.rs

```diff
@@ -518,3 +518,34 @@ fn expression_empty_parentheses() {
         ))
         .stdout(predicate::str::is_empty());
 }
+
+#[test]
+fn find_age_range() {
+    let args = ["-amin", "-cmin", "-mmin"];
+    let times = ["-60", "-120", "-240", "+60", "+120", "+240"];
+    let time_strings = [
+        "\"-60\"", "\"-120\"", "\"-240\"", "\"-60\"", "\"-120\"", "\"-240\"",
+    ];
+
+    for (arg, time) in args.iter().zip(times.iter()) {
```

Comment:
I think this piece of code doesn't do what you think it does ;-) It doesn't create the Cartesian product of `args` and `times`:
```
for (arg, time) in args.iter().zip(times.iter()) {
    println!("{arg}: {time}");
}
```
outputs:
```
-amin: -60
-cmin: -120
-mmin: -240
```

Reply:
Agree. I replaced it with a nested loop. : )